src/app/campaign/flight/flight-form.component.html

```diff
@@ -47,7 +47,7 @@
     <mat-form-field class="flight-form-field" appearance="outline">
       <mat-label>Series</mat-label>
       <mat-select formControlName="set_inventory_uri" required>
-        <mat-option *ngFor="let inv of inventory" [value]="inv.self_uri">{{ inv.podcastTitle }}</mat-option>
+        <mat-option *ngFor="let inv of inventory" [value]="inv.self_uri">{{ inv.podcastId }}: {{ inv.podcastTitle }}</mat-option>
```

Comment:
We display podcast ids elsewhere, so why not here too?

Reply:
Feeder ids are really important and used in a lot of places.  Having a spot to `ctrl-c` and snag them is always nice.